MDANSE_GUI/Src/MDANSE_GUI/Tabs/Models/PlottingContext.py

```diff
@@ -323,7 +327,7 @@ def add_ord(val: Iterable[int]) -> None:
                 )
                 continue
 
-        self._data_limits = list(complete_subset.keys()) if complete_subset else None
+        self._data_limits = list(complete_subset.keys()) if complete_subset else []
```

Comment:
```suggestion
        self._data_limits = list(complete_subset)
```
is all you need in this case.